cpp/src/io/statistics/statistics.cuh

```diff
@@ -114,5 +119,22 @@ struct statistics_merge_group {
   uint32_t num_chunks;           //!< Number of chunks in group
 };
 
+template <typename T, std::enable_if_t<!std::is_same_v<T, statistics::byte_array_view>>* = nullptr>
+__device__ T get_element(column_device_view const& col, uint32_t row)
+{
+  return col.element<T>(row);
+}
+
+template <typename T, std::enable_if_t<std::is_same_v<T, statistics::byte_array_view>>* = nullptr>
+__device__ T get_element(column_device_view const& col, uint32_t row)
+{
+  using et              = typename T::element_type;
+  size_type index       = row + col.offset();  // account for this view's _offset
+  auto const* d_offsets = col.child(lists_column_view::offsets_column_index).data<offset_type>();
+  auto const* d_data    = col.child(lists_column_view::child_column_index).data<et>();
+  size_type offset      = d_offsets[index];
+  return T(d_data + offset, d_offsets[index + 1] - offset);
```

Comment:
```suggestion
  return {(d_data + offset, d_offsets[index + 1] - offset};
```

Reply:
This actually results in a narrowing conversion issue. I can look more into it if you'd like.

Reply:
That's odd.  Should still just be calling the constructor.